bindings/java/pom.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.foundationdb</groupId>
+  <artifactId>fdb-java</artifactId>
+  <version>1.0-SNAPSHOT</version>
```

Comment:
If we end up checking in the `pom.xml` file like this, I agree. But the problem where it uses the wrong version still needs to be addressed. I suspect it may be easier to just continue using a `pom.xml.in` file to generate a `pom.xml`, in which case the `.gitignore` remains the same.